src/resolvers/Query/helperFunctions/getInputArgs.ts

```diff
@@ -0,0 +1,713 @@
+import {
+  DonationWhereInput,
+  EventWhereInput,
+  InputMaybe,
+  OrganizationWhereInput,
+  PostWhereInput,
+  UserWhereInput,
+} from "../../../types/generatedGraphQLTypes";
+
+/**
+ * This function returns FilterQuery object which can be used to find out documents matching specific args as mentioned in `where`.
+ * When modifying this function, check if the arg to be added isn't present before, and place `where` argument
+ * type if not present before in the intersection type.
+ * @typeParam T - Type of object to be returned
```

Comment:
`T` is no longer the type of object to be returned, it is used to return the denote the shape of FilterQuery for an object of shape `T` which this function returns.

Reply:
Done.